cookbooks/proxyfs/files/default/usr/bin/unmount_and_stop_pfs

```diff
@@ -69,7 +72,10 @@ else
     # sudo /usr/sbin/service winbindd stop
     sudo /usr/sbin/service proxyfsd stop
     await_proxyfsd_shutdown
-    sudo /usr/bin/swift-init main stop
+
+    # NOTE: We purposely stop Swift with "swift-init all stop" and not
+    # "swift-init main stop".   See start_and_stop_pfs for more details.
+    sudo /usr/bin/swift-init all stop
     # sudo /usr/sbin/service swift stop
```

Comment:
Can I remove the above line "# sudo /usr/sbin/service swift stop" ?

Reply:
Yes